MANIFEST.in

```diff
@@ -1,27 +1,20 @@
-# Things to always exclude
-prune sphinx
-prune tests
-global-exclude .git*
-global-exclude .ipynb_checkpoints
-global-exclude *.py[co]
-global-exclude __pycache__/**
-
 # Top-level Config
 include CHANGELOG
-include LICENSE.txt
-include MANIFEST.in
+prune tests
 
 # Package files and data
-include bokeh/LICENSE.txt
-include bokeh/py.typed
 graft bokeh/core/_templates
 graft bokeh/sampledata/_data
 graft bokeh/sphinxext/_templates
 graft bokeh/server/static
 exclude bokeh/server/static/js/*.json
 global-exclude *.js.map
+global-exclude *.ts.map
+prune bokeh/server/static/js/types
```

Comment:
@mattpap please confirm whether these two additions are OK

Reply:
OK well I guess at least one of them is needed. Let me know

Reply:
The later is definitely need for extensions compiler. Whereas the former can be removed, because we don't map our sources as far down as TypeScript.

Also, looking at those error messages, it may indicate that extensions' tsconfig files are not well defined:
```
error TS5055: Cannot write file '/usr/share/miniconda3/envs/bk-test/lib/python3.8/site-packages/bokeh/server/static/js/lib/core/bokeh_events.js' because it would overwrite input file.
[PY]   Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
```
Of course, restoring type declarations will help alleviate this, but it's still good to know for future development.